src/components/map/MapUtilsService.js

```diff
@@ -203,7 +203,7 @@ goog.require('ga_urlutils_service');
             duration: 0
           }, function(success) {
             defer.resolve();
-            $rootScope.$applyAsync();
+            $rootScope.$apply();
```

Comment:
why do you remove the '.$applyAsync' ?

Reply:
The tests were failing with `$applyAsync()`